src/Mod/Draft/draftviewproviders/view_layer.py

```diff
@@ -70,16 +70,16 @@ def set_override_options(self, vobj, properties):
                              _tip)
             vobj.OverrideLineColorChildren = True
 
-        if "OverrideShapeAppearanceChildren" not in properties:
```

Comment:
These are the only functional changes. OverrideShapeColorChildren was renamed OverrideShapeAppearanceChildren to more accurately reflect the change in the underlying attribute.